elo_boat.py

```diff
@@ -444,110 +442,104 @@ async def delete_account(ctx):
-                query = f"SELECT elo,elo_convergence,suspended FROM `player` WHERE wc3_name = '{player}'"
-                cursor = my_db.cursor()
-                cursor.execute(query)
-                result = cursor.fetchone()
-                if result is not None:
-                    if result[2] != 0:
-                        await ctx.channel.send("Cannot balance this game : account " + player + " has been suspended")
-                        return
-                    ELO.append([player, trueenv.Rating(result[0], result[1])])
-                else:
-                    # using default ELO
-                    await ctx.send(f"**{player}** was not found, using fresh new player elo for that player")
-                    ELO.append([player, trueenv.Rating(start_elo, start_elo_convergence)])
-            elif "<@!" in player:
-                # mention
-                pass
+    if len(players) % 2 != 0:
+        await ctx.send("Enter a even number of players")
+        return
+
+    if len(list(players)) != len(set(players)):
+        await ctx.send("Duplicate found")
+        return
+
+    for player in players:
+
+        # player is wc3name
+        if "#" in player:
+            query = f"SELECT elo,elo_convergence,suspended FROM `player` WHERE wc3_name = '{player.lower()}'"
+            cursor = my_db.cursor()
+            cursor.execute(query)
+            result = cursor.fetchone()
+            if result is not None:
+                if result[2] != 0:
+                    await ctx.channel.send("Cannot balance this game. Account: **" + player + "** has been suspended")
+                    return
+                ELO.append([player, trueenv.Rating(result[0], result[1])])
             else:
-                # alias
-                player = player.lower()
-                query = f"SELECT wc3_name,elo,elo_convergence,suspended FROM `player` WHERE `alias` = '{player}'"
-                cursor = my_db.cursor()
-                cursor.execute(query)
-                result = cursor.fetchone()
-                if result is None:
-                    await ctx.send(f"**{player}** was not found, using fresh new player elo for that player")
+                await ctx.send(f"**{player}** was not found.")
+                return
 
-                    ELO.append([player, trueenv.Rating(start_elo, start_elo_convergence)])
-                else:
-                    if result[3] != 0:
-                        await ctx.channel.send("Cannot balance this game : account " + result[0] + " has been suspended")
-                        return
-                    print(result)
-                    ELO.append([result[0], trueenv.Rating(result[1], result[2])])
-
-        for mention in ctx.message.mentions:
-            query = "SELECT elo,elo_convergence,wc3_name,suspended FROM `player` WHERE discord_id = " + str(mention.id)
+        # player is a mention
+        elif not (not ("<@" in player) or "<@&" in player):

```

Comment:
woahhhhhhhhh this condition is hard to understand man
so if there is "<@&" it's not a mention but if it's only "<@" it's good i believe right ?

what about replacing this by :
"<@" in player and not("<@&" in player)